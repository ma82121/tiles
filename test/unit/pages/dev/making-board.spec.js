import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vuex from 'vuex'
import MakingBoard from '@/pages/dev/making-board'

import {
  state as configState,
  mutations as configMutations,
  getters as configGetters,
  actions as configActions
} from '@/store/config'
import {
  state as debugState,
  mutations as debugMutations,
  getters as debugGetters,
  actions as debugActions
} from '@/store/debug'

const localVue = createLocalVue()
localVue.use(Vuex)

describe('MakingBoard', () => {
  let store
  beforeEach(() => {
    store = new Vuex.Store({
      modules: {
        config: {
          namespaced: true,
          state: configState,
          mutations: configMutations,
          getters: configGetters,
          actions: configActions
        },
        debug: {
          namespaced: true,
          state: debugState,
          mutations: debugMutations,
          getters: debugGetters,
          actions: debugActions
        }
      }
    })
  })

  test('snapshot', () => {
    const wrapper = shallowMount(MakingBoard, {
      store,
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
