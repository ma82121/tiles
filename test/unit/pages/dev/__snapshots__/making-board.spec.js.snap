// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MakingBoard snapshot 1`] = `
<div
  class="content"
>
  <div>
    <board-stub />
     
    <div>
      <controls-stub
        bg="1"
        boardcount="0"
        boardlogs=""
        stagecurrent="-1"
      />
    </div>
  </div>
   
  <div>
    <div>
      
      Step1 タイルのIDを決定する
    
    </div>
     
    <div
      style="display: none;"
    >
      
      Step2 問題を作成する
    
    </div>
     
    <button
      class="button is-warning"
    >
      
      Step2
    
    </button>
  </div>
   
  <div>
    問題成立 false
  </div>
   
  <div>
    
    Answer
    
    <div>
      []
    </div>
  </div>
</div>
`;
